lib/storage/dbbackend.go

```diff
@@ -1,17 +0,0 @@
-package sebakstorage
-
-type DBBackend interface {
-	Has(string) (bool, error)
-	Get(string, interface{}) error
-	New(string, interface{}) error
-	Set(string, interface{}) error
-	Remove(string) error
-
-	GetIterator(prefix string, reverse bool) (func() (IterItem, bool), func())
-
-	News(...Item) error
-	Sets(...Item) error
-
-	Commit() error
-	Discard() error
-}
```

Comment:
Why do you remove those ? Looks unrelated

Reply:
@Geod24 Oops, it's mistake 👍